internal/servicedeployer/compose.go

```diff
@@ -220,14 +227,18 @@ func (s *dockerComposeDeployedService) TearDown() error {
 		return fmt.Errorf("could not create Docker Compose project for service: %w", err)
 	}
 
+	extraArgs := []string{}
+	if s.shutdownTimeout != "" {
+		extraArgs = append(extraArgs, "-t", s.shutdownTimeout)
+	}
```

Comment:
Created a separate Pull Request for this change:
https://github.com/elastic/elastic-package/pull/1727